src/components/MeasureResults/MeasureResultsRow.js

```diff
@@ -8,50 +8,50 @@ import React from 'react';
 function MeasureResultsRow({ measureResult, handleMeasureChange, measureColor }) {
   return (
     <Box>
-      <Divider color="black" />
-      <Grid container justifyContent="space-evenly" direction="row" alignItems="center" spacing={2} sx={{ width: '100%', p: '3px', m: '2px' }}>
+      <Divider color="#78909C" stroke={2} />
+      <Grid container justifyContent="space-evenly" align-items="center" direction="row" spacing={2} sx={{ width: '100%', p: '3px', m: '0px' }}>
         <Grid item xs={1}>
-          <Typography variant="caption">
+          <FormGroup sx={{ ml: '0' }}>
+            <Checkbox
+              style={{ color: measureColor.color }}
+              disableRipple
+              defaultChecked
+              size="medium"
+              value={measureResult.value}
+              onChange={(event) => handleMeasureChange(event)}
+            />
+          </FormGroup>
+        </Grid>
+        <Grid item xs={1}>
+          <Typography variant="caption" className="measure-results-table__data-measure-title">
             {measureResult.label}
           </Typography>
         </Grid>
         <Grid item xs={2}>
```

Comment:
Give the grid items a class and put a `margin: auto 0` to center the text. Important if we are going to have the longer titles in the data table

Reply:
moving into class now.